src/main/kotlin/coverosR3z/misc/types/Date.kt

```diff
@@ -42,17 +45,20 @@ class Date(val epochDay : Int) : Comparable<Date> {
      */
     val stringValue = java.sql.Date.valueOf(LocalDate.ofEpochDay(epochDay.toLong())).toString()
 
-    private val sdf: SimpleDateFormat = SimpleDateFormat("MMDDYYYY")
+    /**
+     * See here for the description of this formatting: https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html
+     */
+    private val sdf: SimpleDateFormat = SimpleDateFormat("MMddYYYY")
 
     /**
-     * Chrome format "MMDDYYYY"
+     * Chrome format "MMddYYYY"
```

Comment:
found a little bug - uppercase D is the number of the day in the year, not the month